tests/fsdp/test_fsdp.py

```diff
@@ -405,6 +408,8 @@ def test_accelerate_fsdp2_integration(self):
                 self.assertAlmostEqual(log["learning_rate"], log1["learning_rate"], delta=1e-5)
 
     @require_torch_multi_accelerator
+    @require_torch_gpu
```

Comment:
should be removed when https://github.com/huggingface/accelerate/pull/3631 is released